woocommerce/woocommerce
# CYS: Fix CSS header

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This regression has been caused by https://github.com/WordPress/gutenberg/pull/61579. This PR added the CSS necessary that Gutenberg's PR removed.

For this reason, in the near future, we should prioritize [this issue](https://github.com/woocommerce/woocommerce/issues/48377) and get rid of all `edit-site` classes.

Closes #48314.

| Before | After |
|--------|--------|
|![image](https://github.com/woocommerce/woocommerce/assets/4463174/cf8515d0-8e63-4755-bb64-06d92ea7fe35)|![image](https://github.com/woocommerce/woocommerce/assets/4463174/949a4f62-c6a9-4010-ad9d-a3730601399a)| 



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

⚠️ No need to test when testing the release

1. Ensure that you have enabled Gutenberg trunk (last version or at least this commit `97ccc28eae5ac03acf282872c5267419b201e50d`)
2. Head over to WooCommerce > Home > Customize your store.
3. Click on Design Your Homepage.
4. Ensure that the page doesn't look like the image in the `before` column of the table above.
5. Disable Gutenberg.
6. Head over to WooCommerce > Home > Customize your store.
7. Click on Design Your Homepage.
8. Ensure that the page doesn't look like the image in the `before` column of the table above.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message

CYS: Fix CSS header.

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
